python/cudf/cudf/core/series.py

```diff
@@ -1415,31 +1416,52 @@ def as_mask(self):
         """
         return self._column.as_mask()
 
-    def astype(self, dtype, errors="raise", **kwargs):
+    def astype(self, dtype, copy=True, errors="raise", **kwargs):
```

Comment:
Does Pandas take in `**kwargs` here? I see we're using it to pass a format string for datetime casting below but this seems like something that we should just implement `cudf.to_datetime` for.

Reply:
Yes, you are correct. We are exclusively using `**kwargs` to pass format string, Sure will come up with an implementation of `cudf.to_datetime` which will be called from `astype` if the target type is one of `datetime` types.

Reply:
Had a discussion offline and decided we'll have `cudf.to_datetime` added as a follow-up PR.